src/importexport/musicxml/tests/musicxml_tests.cpp

```diff
@@ -47,11 +47,12 @@ extern engraving::Err importCompressedMusicXml(MasterScore*, const QString&);
 
 static const String XML_IO_DATA_DIR("data/");
 
-static const std::string MODULE_NAME("importexport");
+static const std::string MODULE_NAME("iex_musicxml");
 
 static const std::string PREF_EXPORT_MUSICXML_EXPORTBREAKS("export/musicXML/exportBreaks");
 static const std::string PREF_IMPORT_MUSICXML_IMPORTBREAKS("import/musicXML/importBreaks");
 static const std::string PREF_EXPORT_MUSICXML_EXPORTLAYOUT("export/musicXML/exportLayout");
+static const std::string PREF_IMPORT_MUSICXML_IMPORTLAYOUT("import/musicXML/importLayout");
```

Comment:
Not needed (anymore). But probably doesn't harm either

Reply:
Removed, thanks!